src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.jdbc.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
```

Comment:
let's make this class final - because it is an Util class
And also, create a private constructor for this class to prevent initializing the object of util class